### PR TITLE
Add mask to input

### PR DESCRIPTION
**Pull request by eyalella**
added mask option to input using [v-mask](https://www.npmjs.com/package/v-mask)

### CODE REVIEW

File: components/FbxTextField/FbxTextField.stories.js [hunk 1]
```diff
@@ -58,4 +59,34 @@ stories.add('password', () => ({
...
+      action(`New value: ${value}`)()
+    },
+  },
+  template: `
+    <fbx-text-field
+      mask="(###) ###-####"
+      name="email"
+      class="input"
+      validations="required"
+      placeholder="Enter your email"
```
Comment (forforeach) [inline components/FbxTextField/FbxTextField.stories.js]: Enter your phone?

File: components/FbxTextField/FbxTextField.vue [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <div class="fbx-text-field__wrapper">
       <div class="fbx-text-field__input-wrapper">
         <input
+          v-mask="mask"
```
Comment (forforeach) [inline components/FbxTextField/FbxTextField.vue]: How does it behave with empty `mask`?

File: components/FbxTextField/FbxTextField.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 - Use `FbxTextField` for inputs of common types, e.g. `type="text"`, `type="email"`. If your input type isn't supported, open up a PR to add support for that input type
 - The default `type` is `text`, if you don't provide one
+- You can pass a `mask` prop to set up masking. See more info [here](https://www.npmjs.com/package/v-mask)
```
Comment (forforeach) [inline components/FbxTextField/FbxTextField.md]: Add mask property to example as well

File: components/FbxTextField/FbxTextField.vue (hunk 2 above)
Comment (eyalella) [reply components/FbxTextField/FbxTextField.vue]: It does nothing

**Comment by nlitwin** [comment]
For the update phone field in the Fundbox Pay dashboard settings tab, we used `v-mask` on `FbxTextField` directly:
```
   <fbx-text-field
      name="phoneNumber"
      type="tel"
      data-vv-as="phone number"
      validations="min:14"
      :label="'Phone Number (not required)'"
      v-model="phoneNumber"
      v-mask="'(###) ###-####'"
      data-qa="phone-number-input"
    />
```

But adding a mask prop works too!

**Pull request merged**